platform/metrics.go

```diff
@@ -26,28 +27,28 @@ func NewMetrics() Metrics {
 			Name:      "all_services_duration_seconds",
 			Help:      "AllServices method duration in seconds.",
 			Buckets:   stdprometheus.DefBuckets,
-		}, []string{"namespace", "success"}),
+		}, []string{"namespace", fluxmetrics.LabelSuccess}),
```

Comment:
I'd rather either, all the labels live in fluxmetrics, or none. The half-half thing is weird.